web/modules/custom/employees/employees.module

```diff
@@ -128,6 +129,37 @@ function employees_form_alter(&$form, &$form_state, $form_id)
       }
       break;
   }
+
+  // PE-595 Add a custom submit handler to the Resource form submit action to alter the redirect URL
+  if ($form_id === 'node_resource_form' || $form_id === 'node_resource_edit_form')  {
+    foreach (array_keys($form['actions']) as $action) {
+      if ($action != 'preview' && isset($form['actions'][$action]['#type']) && $form['actions'][$action]['#type'] === 'submit') {
+        $form['actions'][$action]['#submit'][] = '_resource_node_form_submit';
```

Comment:
Could this be simplified to `$form['actions']['submit']['#submit'][]  = '_resource_node_form_submit';` without the for loop and associated if statement, since we're just trying to overwrite the main save/submit button?

Also, perhaps we should combine these form_ids as cases in the big switch statement above? Although I kind of hate switch statements, it would be consistent with the existing code 😆 

Otherwise, looks good!